src/rumo_sdk/content_api.py

```diff
@@ -26,10 +26,10 @@ def get_items(
             params.update(rumo_filters.format_filters_to_query_params())
         return self._rumo_client.get(endpoint, query_params=params)
 
-    def get_items_id(self) -> dict:
+    def get_items_id(self, **kwargs) -> dict:
```

Comment:
sorry about the inconsistency in the library for `kwargs`, I only added them when I needed to use them :confused: